evm_loader/program/src/solidity_account.rs

```diff
@@ -1,27 +1,30 @@
-use crate::hamt::Hamt;
 use crate::account_data::AccountData;
-use solana_sdk::{
-    account_info::AccountInfo,
-    program_error::ProgramError,
-    pubkey::Pubkey,
-};
+use solana_sdk::program_error::ProgramError;
```

Comment:
Please do not make unneeded changes. Moreover, in my opinion, it's better to see which modules are used from solana_sdk in one place.